src/flags/general.cpp

```diff
@@ -131,6 +131,14 @@ DEFINE_uint64(storage_recovery_thread_count,
 DEFINE_bool(storage_enable_schema_metadata, false,
             "Controls whether metadata should be collected about the resident labels and edge types.");
 
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_bool(storage_enable_automatic_label_index_creation, false,
+            "Controls whether label indexes on vertices should be created automatically.");
+
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_bool(storage_enable_automatic_edge_type_index_creation, false,
+            "Controls whether edge-type indexes on relationships should be created automatically.");
```

Comment:
let's change the format to be aligned with the rest of the flags (put `enabled` at the end), so e.g. `storage_automatic_label_index_creation_enabled` (the same for the other flag)